package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "clean": "rm -rf dist",
     "compile": "node -r dotenv/config --harmony bin/compile",
-    "lint": "eslint . ./",
+    "lint": "eslint --est=js --est=jsx . ./",
```

Comment:
@CurtisHumphrey 
I am thinking - wouldn't it be better if stuff like that be configurable?